.changeset/fresh-pears-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+'@prairielearn/postgres': minor
+---
+
+Deprecate queryCursor
```

Comment:
"Deprecate" generally means "label as obsolete and not recommended for use"; you're removed it outright.


```suggestion
---
'@prairielearn/postgres': major
---

Remove `queryCursor`
```